frontend/src/components/uam/helpers/get-columns-helper/get-groups-columns.helper.ts

```diff
@@ -16,6 +18,13 @@ const getGroupsColumns = (): Column<GroupsItemResponseDto>[] => {
       Header: 'Key',
       accessor: GroupsTableAccessor.KEY,
     },
+    {
+      Header: 'Actions',
+      accessor: ({ id }: GroupsItemResponseDto): GroupsTableActionsProps => ({
+        id,
+      }),
```

Comment:
```suggestion
      accessor: GroupsTableAccessor.ID,
```
if you don't plan to add new props 🤔

Reply:
New props are being added in delete functionality, decided to do it this way, so that there are not a lot of changes to make

Reply:
Got it 👍